### ISSUE TITLE
Wheel size if ~ 8x bigger

### ISSUE THREAD

**Issue by pombredanne** [nexB/cwe2#15]
https://pypi.org/project/cwe2/3.0.0/#files ... the wheel are much bigger than before.
Is this expected?
@ziadhany ?

**Comment by ziadhany** [comment]
@pombredanne Yes, we have updated the database to include all CWE information and ensured that no CWE data is missing.

You can track these updates by following the mapping in the link below:
https://github.com/nexB/cwe2/blob/main/src/cwe2/mappings.py

Here is a concise analysis of the overall database differences between the two releases:

### V2 database:  total 5.4 MB

      
    - 2.6M 1000.csv
    - 290K 1194.csv
    - 825K 1344.csv
    - 288K 1387.csv
    - 1.2M 699.csv


### V3 database: total 48.3 MB


        - 14M cwec_v4.14.xml

-  external_mappings/ total 7.4M

        - 965K 1008.csv
        - 428K 1133.csv
        - 423K 1154.csv
        - 145K 1178.csv
        - 582K 1305.csv
        - 575K 1340.csv
        - 54K 1343.csv
        - 909K 1344.csv
        - 397K 1358.csv
        - 265K 1424.csv
        - 308K 1425.csv
        - 428K 700.csv
        - 2.1M 888.csv

-  helpful_views/ total 15M

        - 880K 1003.csv
        - 122K 1040.csv
        - 624K 1081.csv
        - 2.9M 1400.csv
        - 2.9M 2000.csv
        - 8.7K 604.csv
        - 349K 658.csv
        - 358K 659.csv
        - 200K 660.csv
        - 124K 661.csv
        - 1.6M 677.csv
        - 22K 678.csv
        - 1.1M 701.csv
        - 2.5M 702.csv
        - 6.5K 709.csv
        - 843K 884.csv
        - 123K 919.csv

-  navigate_cwe: total 4.5M

        - 2.9M 1000.csv
        - 361K 1194.csv
        - 1.3M 699.csv

-  obsolete_views: total 6.0M

        - 280K 1026.csv
        - 256K 1128.csv
        - 309K 1200.csv
        - 304K 1337.csv
        - 316K 1350.csv
        - 309K 1387.csv
        - 247K  629.csv
        - 203K 635.csv
        - 604K 711.csv
        - 507K 734.csv
        - 314K 750.csv
        - 424K 800.csv
        - 330K 809.csv
        - 456K 844.csv
        - 564K 868.csv
        - 437K 900.csv
        - 263K 928.csv
